### PR TITLE
Set up test environment DONOR

### PR DESCRIPTION
**Pull request by FlisAnn**
[Set up test environment DONOR](https://www.pivotaltracker.com/story/show/176623611)
### User Story
```
As a development team,
In order to unit test our code
We would like to set up Cypress
```
## Changes proposed in this pull request:
* Setting up Cypress
* Downgrading to Cypress 4.1.0
## What I have learned working on this feature:
* How to install Cypress
## Packages/Gems added
* Cypress

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (swardbladpaulo) [inline cypress/fixtures/example.json]: Please remove this example

File: cypress/integration/test.feature.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  beforeEach(() => {
+    cy.visit("/")
+    
+  });
+  
+  
+  it('finds root', () => {
+    expect("contain", "weCare")
+  });
+})
```
Comment (swardbladpaulo) [inline cypress/integration/test.feature.js:8]: Format code.

File: cypress/fixtures/example.json (hunk 1 above)
Comment (FlisAnn) [reply cypress/fixtures/example.json]: It has been removed

File: cypress/integration/test.feature.js (hunk 2 above)
Comment (FlisAnn) [reply cypress/integration/test.feature.js:8]: Files is formatted

**Pull request merged**